src/Glue.elm

```diff
@@ -1,24 +1,24 @@
 module Glue exposing
     ( Glue
-    , glue, simple, poly
+    , simple, poly, glue
```

Comment:
I think we can start with the simpler ones and go to more general? `simple` :arrow_right: `poly` :arrow_right: `glue`

Reply:
this is how it is an old version. Anyway, I think now with changes `glue` is not really that hard to get. Also `simple` is not really that useful. I've added it mostly for more compatibility with `Browser.sandbox` (kind of makes sense to offer something). But don't change it yet. Those are just first thoughts. I need to process it a bit more.

Reply:
ok, let's keep it as is... simple first, glue last.